webapp/app/model/eligibility_data.py

```diff
@@ -483,25 +432,28 @@ class ForeignCountrySuccessEligibility(RecursiveDataModel):
     """
     has_no_other_income: Optional[OtherIncomeEligibilityData]
     foreign_country_eligibility: str
-    elster_registration_method_eligibility: Optional[str]
-    elster_abrufcode_eligibility: Optional[str]
+    user_a_has_elster_account_eligibility: str
+    user_b_has_elster_account_eligibility: Optional[str]
+    
+    @validator('user_b_has_elster_account_eligibility', always=True)
+    def has_user_b_elster_account_eligibility(cls,v, values):
+        user_a = values.get('user_a_has_elster_account_eligibility')
+        user_b = v
+        
+        try:
+            declarations_must_be_set_no(user_a)
+        except:
+            if user_b:
+                return declarations_must_be_set_no(user_b)
```

Comment:
In case it's not clear to anyone:

* If User A has no Elster account & User B has no Elster account = continue.
* If User A has no Elster account & User B does not exist = continue
* If User A has no Elster account & User B has an Elster account = Maybe page
* If User A has an Elster account & User B has an Elster account = Maybe page